aiohttp/helpers.py

```diff
@@ -244,6 +243,31 @@ class ProxyInfo:
     proxy_auth: Optional[BasicAuth]
 
 
+def basicauth_from_netrc(netrc_obj: netrc.netrc, host: str) -> Optional[BasicAuth]:
+    auth_from_netrc = netrc_obj.authenticators(host)
+    if auth_from_netrc is not None:
+        login, account, password = auth_from_netrc
+        if not login:
+            # login is provided, we just use it as username.
```

Comment:
```suggestion
        if login:
            # login is provided, we just use it as username.
```

Think that's reversed?